packages/console/src/pages/Applications/components/GetStarted/index.tsx

```diff
@@ -98,42 +143,69 @@ const GetStarted = ({ appName, onClose }: Props) => {
       </div>
       <div className={styles.content}>
         {isLibrarySelectorFolded ? librarySelectorFolded : librarySelector}
-        {/* TO-DO: Dynamically render steps from markdown files */}
-        <Card className={styles.card}>
-          <div className={styles.index}>1</div>
-          <CardTitle
-            title="applications.get_started.title_step_1"
-            subtitle="applications.get_started.subtitle_step_1"
-          />
-        </Card>
-        <Card className={styles.card}>
-          <div className={styles.index}>2</div>
-          <CardTitle
-            title="applications.get_started.title_step_2"
-            subtitle="applications.get_started.subtitle_step_2"
-          />
-        </Card>
-        <Card className={styles.card}>
-          <div className={styles.index}>3</div>
-          <CardTitle
-            title="applications.get_started.title_step_3"
-            subtitle="applications.get_started.subtitle_step_3"
-          />
-        </Card>
-        <Card className={styles.card}>
-          <div className={styles.index}>4</div>
-          <CardTitle
-            title="applications.get_started.title_step_4"
-            subtitle="applications.get_started.subtitle_step_4"
-          />
-        </Card>
-        <Card className={styles.card}>
-          <div className={styles.index}>5</div>
-          <CardTitle
-            title="applications.get_started.title_step_5"
-            subtitle="applications.get_started.subtitle_step_5"
-          />
-        </Card>
+        {steps?.map((step, index) => {
```

Comment:
Complicated element rendering logic could extract to a useMemo ?

Reply:
I would probably extract some components out of this big one, as the get-started is also required in connectors. By then, there would probably be a list of <Step> components.